pkgs/tools/filesystems/dwarfs/default.nix

```diff
@@ -90,17 +91,31 @@ stdenv.mkDerivation {
   ];
 
   doCheck = true;
-  nativeCheckInputs = [ gtest ];
+  nativeCheckInputs = [
+    # https://github.com/mhx/dwarfs/issues/188#issuecomment-1907574427
+    # `dwarfs` sets C++20 as the minimum; thus the `gtest` headers, when included,
+    # refer to symbols that only exist in `.so` files compiled with that version.
+    (gtest.override { cxx_standard = "20"; })
```

Comment:
This situation is kind of hokey, I guess it was too annoying to download gtest as a separate thing and move it into the build? That can be very particular with CMake.

Reply:
Yes, and this way at least when we patch `gtest` to work well in nixpkgs, we at least dont have to do it again in this package.